src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/RemoveInternetSourcesFromNuGetConfig.cs

```diff
@@ -35,15 +35,40 @@ public class RemoveInternetSourcesFromNuGetConfig : Task
         /// </summary>
```

Comment:
Since you are adding functionality, please consider adding #nullable enable to this file.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/42740/commits/fadcef88b4c349cb646af94b7db9e2f1e2a756bc